pkg/metrics/metrics.go

```diff
@@ -140,6 +140,26 @@ func InitializeMetrics() {
 		dnsResponseCounterDescription,
 	)
 
+	// Conntrack Metrics
+	ConntrackPacketsCounter = exporter.CreatePrometheusGaugeVecForMetric(
```

Comment:
Can you find a way to integrate these into Hubble metrics as well ?

Reply:
still WIP, but some initial work is done, see this screenshot
![image](https://github.com/user-attachments/assets/e474adec-2a45-42a2-9f85-237b9feb2e16)